site/content/docs/5.2/forms/layout.md

```diff
@@ -212,6 +233,13 @@ As shown in the previous examples, our grid system allows you to place any numbe
 
 The example below uses a flexbox utility to vertically center the contents and changes `.col` to `.col-auto` so that your columns only take up as much space as needed. Put another way, the column sizes itself based on the contents.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This form variant can not be used because it does not respect the Orange Design System specifications. The form text input label should always be placed above the input field.
```

Comment:
For what I see, there's only one visible label. I think the explanation must be rethought here. + These form variants

Reply:
OK to write "These form variants".

I don't understand "For what I see, there's only one visible label. I think the explanation must be rethought here"...

Reply:
Message for case when label not at the right place:
This form variant with labels not placed above input fields can not be used because it does not respect the Orange Design System specifications. The form text input label should always be placed above the input field.

For the correct design, please refer to [Forms](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459) and [Pages](https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53) on the Orange Design System website.

Message for case when labels missing in form:  TO WORK ON !!!
This form variant without label input fields can not be used because it does not respect the Orange Design System specifications. The form text input label should always be placed above the input field.

For the correct design, please refer to [Forms](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459) and [Pages](https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53) on the Orange Design System website.